entrypoint.sh

```
@@ -17,12 +17,19 @@ LOGS_DIR=${LOGS_DIR:-"/data/logs"}
 TEMP_DIR=${TEMP_DIR:-"/data/temp"}
+SKIP_CHOWN=${SKIP_CHOWN:-false}
 
 # set permissions
-chown -R abc:abc ${LOGS_DIR}
-chown -R abc:abc ${CONFIG_DIR}
-chown -R abc:abc ${TEMP_DIR}
-chown abc:abc ${VIDEOS_DIR}
+# SKIP_CHOWN can be set to true to skip this, useful for network storage (NFS/SMB)
+# that does not support chown or POSIX permissions
+if [ "${SKIP_CHOWN}" != "true" ]; then
+  chown -R abc:abc ${LOGS_DIR}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)